src/parser/core/modules/BuffWindow.tsx

```diff
@@ -172,7 +177,8 @@ export abstract class BuffWindowModule extends Module {
 	}
 
 	private onApplyBuff(event: BuffEvent) {
-		if (!this.buffStatus || event.ability.guid !== this.buffStatus.id) {
+		if ((!this.buffStatus || event.ability.guid !== this.buffStatus.id) &&
```

Comment:
Per discussion [on Discord](https://discord.com/channels/441414116914233364/441424599310270464/793185644699582494) - we should refactor this to use the utilities class method `ensureArray` and change buffStatus to accept `Status | Status[]`

I POC'd this with:
```
	private onApplyBuff(event: BuffEvent) {
		if (!this.buffIsTracked(event.ability.guid)) {
			return
		}

		this.startNewBuffWindow(event.timestamp)
	}

	private onRemoveBuff(event: BuffEvent) {
		if (!this.buffIsTracked(event.ability.guid)) {
			return
		}

		if (this.activeBuffWindow) {
			this.activeBuffWindow.end = event.timestamp
		}
	}

	protected buffIsTracked(buffId: number) {
		let buffStatusArray = ensureArray(this.buffStatus)
		return buffStatusArray.some(bs => bs.id == buffId)
	}

	protected trackedBuffDuration() {
		let buffStatusArray = ensureArray(this.buffStatus)
		return buffStatusArray[0].duration
	}
```

and it seemed to WOMM.